packages/cubejs-mongobi-driver/driver/MongoBIDriver.js

```diff
@@ -0,0 +1,126 @@
+const mysql = require('mysql2');
+const genericPool = require('generic-pool');
+const { promisify } = require('util');
+const BaseDriver = require('@cubejs-backend/query-orchestrator/driver/BaseDriver');
+
+class MongoBIDriver extends BaseDriver {
+  constructor(config) {
+    super();
+    this.config = {
+      host: process.env.CUBEJS_DB_HOST,
+      database: process.env.CUBEJS_DB_NAME,
+      port: process.env.CUBEJS_DB_PORT,
+      user: process.env.CUBEJS_DB_USER,
+      password: process.env.CUBEJS_DB_PASS,
+      ssl: {
+	      ssl: process.env.USE_SSL
```

Comment:
I guess for naming consistency we should use `CUBEJS_DB_SSL` here instead or something like it.